crates/wac-parser/wac.pest

```diff
@@ -85,6 +84,14 @@ InterfaceRef         =  { PackagePath | Ident }
 ExternType           = !{ FuncType | InlineInterface | Ident }
 InlineInterface      =  { InterfaceKeyword ~ InterfaceBody }
 
+WorldIncludeStatement  = ${ IncludeKeyword ~ DelimitingSpace+ ~ WorldIncludeItem ~ DelimitingSpace* ~ Semicolon }
+WorldIncludeItem       = !{ WorldRef ~ DelimitingSpace* ~ WorldIncludeWithClause? }
+WorldIncludeWithClause = ${ WithKeyword ~ DelimitingSpace* ~ WorldIncludeWithBody }
+WorldIncludeWithBody   = ${ OpenBrace ~ DelimitingSpace* ~ IncludeNamesList ~ DelimitingSpace* ~ CloseBrace }
+IncludeNamesList       = ${ IncludeNamesItem ~ DelimitingSpace* ~ ("," ~ DelimitingSpace* ~ IncludeNamesItem ~ DelimitingSpace*)* }
```

Comment:
Likewise, I don't think this one needs to be atomic and should support a tailing `,`:

```suggestion
IncludeNamesList       = !{ IncludeNamesItem ~ ("," ~ IncludeNamesItem)* ~ ","? }
```

Reply:
Also, I think we can just collapse this rule into `WorldIncludeWithBody`, should cut down slightly on the AST structures.